libnavigation-core/src/main/java/com/mapbox/navigation/core/accounts/Billing.kt

```diff
@@ -38,9 +38,9 @@ internal class Billing private constructor() {
 
         private fun setBillingType(context: Context) {
             val applicationInfo =
-                getApplicationInfo(
-                    context
-                )
+                    getApplicationInfo(
```

Comment:
Could we join this four lines in into one line?

Reply:
Done